test/streams.js

```diff
@@ -45,3 +47,66 @@ describe('strong-remoting', function() {
       .expect({bar: 'baz'}, done);
   });
 });
+
+describe('a function returning a ReadableStream', function() {
+  var Readable = require('stream').Readable;
+  var remotes = RemoteObjects.create();
+  var streamClass;
+  var server;
+
+  before(function(done) {
+    var app = express();
+    server = app.listen(0, '127.0.0.1', done);
+    function StreamClass() {
+
+    }
+
+    StreamClass.createStream = function createStream(cb) {
+      var rs = new Readable({objectMode: true});
+      var buf = [];
+      var n = 5;
+      while (n--) {
+        buf.push({n: n});
+      }
+      rs._read = function() {
+        setTimeout(function() {
+          rs.push(buf.shift() || null);
+        }, 1);
+      };
```

Comment:
Good idea. The `setTimeout` is to add some delay, but I don't think its strictly necessary.